ddht/v5/client.py

```diff
@@ -0,0 +1,96 @@
+import logging
+
+from async_service import Service
+from eth_enr import ENRDatabaseAPI
+from eth_keys import keys
+import trio
+
+from ddht.base_message import AnyInboundMessage, AnyOutboundMessage
+from ddht.datagram import (
+    DatagramReceiver,
+    DatagramSender,
+    InboundDatagram,
+    OutboundDatagram,
+)
+from ddht.v5.channel_services import (
+    InboundPacket,
+    OutboundPacket,
+    PacketDecoder,
+    PacketEncoder,
+)
+from ddht.v5.message_dispatcher import MessageDispatcher
+from ddht.v5.messages import v5_registry
+from ddht.v5.packer import Packer
+
+
+class Client(Service):
+    logger = logging.getLogger("ddht.Client")
+
+    def __init__(
+        self, local_private_key: keys.PrivateKey, enr_db: ENRDatabaseAPI, node_id, sock
+    ) -> None:
+
+        self.enr_db = enr_db
+
+        outbound_datagram_channels = trio.open_memory_channel[OutboundDatagram](0)
```

Comment:
If you play with the concurrency levels or identifying bottlenecks, I would suggest tinkering with these channel buffer sizes.